packages/vscode-extension/package.json

```diff
@@ -829,6 +829,71 @@
         "scopeName": "teamsfx-toolkit.output",
         "path": "./syntaxes/teamsfx-toolkit-output.tmLanguage"
       }
+    ],
+    "walkthroughs": [
+      {
+        "id": "teamsToolkitQuickStart",
+        "title": "Get started with Teams toolkit  ",
+        "description": "Jumpstart your Teams app development experience   ",
+        "steps": [
+          {
+            "id": "teamsToolkitCapailities",
+            "title": "What are Teams app \"Capabilities\"",
+            "description": "[Capabilities](https://aka.ms/teamsfx-capabilities-overview) are the extension points for building apps on the Microsoft Teams platform.\n[Watch video (<1 min)](https://aka.ms/teamsfx-video)",
+            "media": {
+              "svg": "media/learn.svg",
+              "altText": "Watch"
+            }
+          },
+          {
+            "id": "teamsToolkitCommands",
+            "title": "Explore Teams toolkit commands",
+            "description": "Open Command Palette (Ctrl+Shift+P) and type 'Teams' to find Teams toolkit commands.\n[Display Teams commands](command:workbench.action.quickOpen?%5B%22%3ETeams%22%5D)",
+            "media": {
+              "svg": "media/step_commands.svg",
+              "altText": "Display Teams Commands"
+            },
+            "when": "!isMac"
```

Comment:
When did you set the isMac context?

Reply:
isMac is a global content key in vs code.